site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
+   - `Radio` enables the selection of one option from several options.
+   - `Checkbox` enables the selection of multiple options.
+   - `Text` sets the attribute to a text field.
+   - `Number` sets the attribute to numerical field in the following format: `min;max;step`.
+4. In the **Attribute values** field, add attribute values. <br>To separate values use **Enter**.
+   <br>To delete value, use **Backspace** or click **x** next to the value name.
+5. (Optional) For mutable attributes, select **Mutable**.
 
-### Chunk size
+To delete an attribute, click **Delete attribute**.
 
-Defines a number of frames to be packed in a chunk when send from client to server.
-Server defines automatically if empty.
+### Select files
 
-Recommended values:
+There are several ways to upload files:
 
-- 1080p or less: 36
-- 2k or less: 8 - 16
-- 4k or less: 4 - 8
-- More: 1 - 4
+<!--lint disable maximum-line-length-->
 
-### Dataset Repository
+| Data source          | Description                                                                                                                                                                                                                                         |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| My computer          | Use this option to select files from your laptop or PC. <br> To select file: <br>1. Click on the **Select files** field: <br>![Select files](/images/select_files.jpg). <br> 2. Select files to upload.                                             |
+| Connected file share | **Advanced option**. <br>Upload files from a local or cloud shared folder. <br>**Note**, that you need to mount a fileshare first. <br>For more information, see [Share path](/docs/administration/basics/installation/#share-path)                 |
+| Remote source        | **Advanced option**.<br>Enter a list of URLs (one per line) in the field. <br>If you want a `manifest.json` file, select the **Use cache**. <br> For more information, see [Dataset manifest.](/docs/manual/advanced/dataset_manifest/)             |
+| Cloud Storage        | **Advanced option**. <br>To upload files from cloud storage, type the cloud storage name, choose the manifest file, and select the required files. <br> For more information, see [Attach cloud storage](/docs/manual/basics/attach-cloud-storage/) |
 
-URL link of the repository optionally specifies the path to the repository for storage
-(`default: annotation / <dump_file_name> .zip`).
-The .zip and .xml file extension of annotation are supported.
-Field format: `URL [PATH]` example: `https://github.com/project/repos.git [1/2/3/4/annotation.xml]`
+<!--lint enable maximum-line-length-->
 
-Supported URL formats :
+### RAW
```

Comment:
I don't think that RAW is the right name here (even if you can see it in UI). Labels are specified as a JSON. You can edit JSON directly, but it is for an advanced use case. Also we have a label constructor. It simplifies the process of specifying labels.

Could you please find an appropriate name? `Editing labels in RAW format` or something like that.